activations/activations.go

```diff
@@ -97,6 +97,19 @@ func (a *Activation[T]) ValuesInFunction() map[string]T {
 	return values
 }
 
+// ValuesInCurrentLevel returns all values in the current activation level.
+func (a *Activation[T]) ValuesInCurrentLevel() map[string]T {
+	values := make(map[string]T)
+
+	for name, value := range a.entries { //nolint:maprange
+		if _, ok := values[name]; !ok {
+			values[name] = value
+		}
+	}
+
+	return values
```

Comment:
Could this just return `a.entries`, does it have to defensively create a copy?

Reply:
Yes, it can! Simplified in https://github.com/onflow/cadence/pull/4009/commits/6503753b96c53b218fe3e4286971d858837dba5a